src/components/Tags/TagAssignmentSheet.tsx

```diff
@@ -125,15 +127,47 @@ export default function TagAssignmentSheet({
 
   const entityConfig = ENTITY_CONFIG[entityType];
 
+  const isFacilityTag = (tag: TagConfig) => {
+    return !!tag.facility;
+  };
+
+  const getTagMutationConfig = (
+    tag: TagConfig,
+    operation: "set" | "remove",
+  ) => {
+    const isFacility = entityType === "patient" && isFacilityTag(tag);
```

Comment:
```suggestion
    const isFacilityPatientTag = entityType === "patient" && isFacilityTag(tag);
```

Just to be clearer 👍  Otherwise might be confusing on first read

Reply:
will change that

Reply:
https://github.com/ohcnetwork/care_fe/pull/14927/commits/0a33fb0112973b9f3ceb6b281cb667e558f80f74 fixed